packages/webapp/components/layouts/AccountLayout/common.tsx

```diff
@@ -117,12 +119,22 @@ export const accountPage: Record<AccountPage, AccountPageProps> = {
       />
     ),
   },
+  invite: {
+    title: 'Invite friends',
+    href: '/invite',
+    getIcon: ({ isActive }) => (
+      <InviteIcon
+        secondary={isActive}
+        className={!isActive && 'text-theme-label-secondary'}
+      />
+    ),
+  },
 };
 
 export const AccountPageContent = classed(
   'div',
   pageBorders,
-  'flex flex-col w-full laptop:max-w-[calc(100vw-19.75rem)] laptop:max-w-[calc(100vw-36rem)] laptopL:max-w-[40rem] tablet:border-l mr-auto',
+  'flex flex-col w-full laptop:max-w-[calc(100vw-19.75rem)] laptopL:max-w-[40.5rem] tablet:border-l mr-auto',
```

Comment:
There was an 8px discrepancy after checking in Sketch. Also removed the unused class.